src/app_service/service/accounts/service.nim

```diff
@@ -405,8 +405,6 @@ QtObject:
       "TorrentDir": DEFAULT_TORRENT_CONFIG_TORRENTDIR
     }
 
-    result["LogLevel"] = newJString(toStatusGoSupportedLogLevel(main_constants.LOG_LEVEL))
```

Comment:
I am removing this line because by doing so, we are forcing the loglevel are login to be the default one, while it should be the last changed value from database or disable the UI naturally if ENV or CLI variable is set. Currently performing more tests to confirm this change but initial tests I made look okay to me. @igor-sirotin in parallel of my tests, any insights dependencies on this line please?

Reply:
Looking into this.

Reply:
Looks like it should be like this:
```nim
if !constants.runtimeLogLevelSet():
    result["LogLevel"] = newJString(toStatusGoSupportedLogLevel(main_constants.LOG_LEVEL))
```

wdyt?

Reply:
ok, probably not that simple

Reply:
Actually my idea looks correct. This will prevent the override of the value stored in the database.

We call `getLoginNodeConfig` and pass it to `login` here:
https://github.com/status-im/status-desktop/blob/9fc5c1f5146f4f9b07da96db50901d54d142a318/src/app_service/service/accounts/service.nim#L678-L679

Which calls this endpoint:
https://github.com/status-im/status-go/blob/69948a7024df8c4537fc4986b39aded22e89179b/mobile/status.go#L216

And falls into `startNodeWithAccount` to this line:
https://github.com/status-im/status-go/blob/69948a7024df8c4537fc4986b39aded22e89179b/api/geth_backend.go#L715

Which will load the config from database and override with values in the config you pass from Nim.
https://github.com/status-im/status-go/blob/69948a7024df8c4537fc4986b39aded22e89179b/api/geth_backend.go#L1691

So we should just prevent pushing it there if the value is not set in runtime.

Reply:
Hey!, agree with that flow, however, you can see that 

if `runtime Log level` is not set, then the default values will kick in, and will be applied here. Making the values always be the default values at each restart of the application.

Do you agree?

That is essentially the reason why this ticket is created in the first place.